### PR TITLE
[CO-1261] Updated documentations for catalog APIs to support JSON data type

### PR DESCRIPTION
**Pull request by shihanxiong**
https://jira.braze.com/browse/CO-1261

### CODE REVIEW

File: _docs/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md [hunk 1]
```diff
@@ -121,6 +125,12 @@ The status code `201` could return the following response body.
           "name": "Loyalty_Program",
           "type": "boolean"
         },
+        {
+          "location": {
+            "Latitude": 33.6112,
+            "Longitude": -117.8711
+          }
+        },
```
Comment (raghavnarain3) [inline _docs/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:133]: I don't believe this is correct

Comment (davehensley) [reply _docs/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:133]: Yeah, this should just be the name and type ("object").

Comment (shihanxiong) [reply _docs/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:133]: Oh good catch

File: _docs/_api/endpoints/catalogs/catalog_items/asynchronous/patch_catalog_items_bulk.md [hunk 2]
```diff
@@ -48,6 +48,10 @@ curl --location --request PATCH 'https://rest.iad-03.braze.com/catalogs/restaura
       "id": "restaurant1",
       "Name": "Restaurant",
       "Loyalty_Program": false,
+      "location": {
```
Comment (davehensley) [inline _docs/_api/endpoints/catalogs/catalog_items/asynchronous/patch_catalog_items_bulk.md:51]: Super nit: "Location" should probably be capitalized like the other non-id fields.

**Review by davehensley — APPROVED** [review]

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**